src/main/resources/db/migration/metadata/V2__update_idl_types.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TYPE idl RENAME TO idl_old;
+
+CREATE TYPE idl as ENUM('avro', 'protobuf', 'mu', 'openapi', 'openapijson', 'scala');
+
+ALTER TABLE metaprotocols ALTER COLUMN idl_name TYPE idl USING idl_name::text::idl;
+
+DROP TYPE idl_old;
```

Comment:
Rethinking about this, we could just have the following:
```suggestion
ALTER TYPE idl ADD VALUE 'openapijson';
```